themes/default/tasks/rotate_comment_readlog.pl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/perl -w
+# This code is a part of Slash, and is released under the GPL.
+# Copyright 1997-2005 by Open Source Technology Group. See README
+# and COPYING for more information, or see http://slashcode.com/.
+# $Id$
+
+use strict;
+use utf8;
+use Slash;
+use Slash::Constants ':slashd';
+use vars qw( %task $me %redirects );
+
+$task{$me}{timespec} = '30 1 * * *';
+$task{$me}{timespec_panic_1} = ''; # not that important
+$task{$me}{fork} = SLASHD_NOWAIT;
+$task{$me}{code} = sub {
+	my($virtual_user, $constants, $slashdb, $user, $info, $gSkin) = @_;
+
+	my $days = $slashdb->sqlSelect('value', 'vars', "name = 'comment_read_max_age'")
```

Comment:
how about 
my days = int($constants->{comment_read_max_age}) || 14;

getting the constants vars already from slashd. no need to call the db again.